src/components/ImportRecordsFlow/helpers/__test__/getDataFromWorkbook.spec.js

```diff
@@ -0,0 +1,15 @@
+import XLSX from 'xlsx';
+import getArrayDataFromWorkBook from '../getDataFromWorkbook';
+import getWorkbookParams from './getWorkbookParams';
+
+XLSX.utils.sheet_to_row_object_array = jest.fn();
+
+describe('getArrayDataFromWorkBook', () => {
```

Comment:
same ^^ here we need to be explicit about what we are testing, then use `getDataFromWorkbook` instead